src/contracts/interfaces/IStrategyManager.sol

```diff
@@ -127,6 +127,9 @@ interface IStrategyManager {
     /// @notice Returns the address of the `strategyWhitelister`
     function strategyWhitelister() external view returns (address);
 
+    /// @notice Returns bool for whether or not `strategy` is whitelisted for deposit
+    function strategyIsWhitelistedForDeposit(IStrategy strategy) external view returns (bool);
+
```

Comment:
How we handle this wasn't part of the design or the use cases. What's the point of adding this? If the strategy is invalid the rows in the pipeline come out as zero. Is this to prevent bad payments from being made?

Reply:
This is just input sanitization. We can assume offchain that we can read staker/operator shares for whitelisted strats and calculate payments proportionally.

Reply:
We aligned offline here, but we basically want to prevent bad payments being made. Eg. AVS sends a 6 figure+ payment to a non-existent strategy and we then have to update the contract to claw back these funds.